UnitTests/Features/Event/MakePublic/MakeEventPublicTests.cs

```diff
@@ -0,0 +1,73 @@
+using ViaEventAssociation.Core.Domain.Aggregates.Events;
+
+namespace UnitTests.Features.Event.MakePublic;
+using Xunit;
+
+public class MakeEventPublicTests
+{
+
+    [Fact]
+    public void MakeEventPublic_StatusIsDraft_ReturnsSuccess()
+    {
+        // Arrange
+        var id = EventId.Create(Guid.NewGuid().ToString());
```

Comment:
Instead of setting up the event instance here, please try to use EventFactory.cs class (currently in UC3-Lukas branch, waiting for re-review)

Reply:
Same goes for all the test cases